types/fabric/fabric-impl.d.ts

```diff
@@ -1953,6 +1953,13 @@ export class Canvas {
      * @param [options] Options object
      */
     constructor(element: HTMLCanvasElement | string | null, options?: ICanvasOptions);
+    /**
+     * Constructor
+     * @param {HTMLCanvasElement | String} element <canvas> element to initialize instance on
+     * @param {Object} [options] Options object
+     * @return {Object} thisArg
+     */
+     initialize(element: HTMLCanvasElement | string | null, options?: ICanvasOptions): Canvas;
```

Comment:
This line is indented with 5 spaces instead of 4; your editor probably tried to mirror the previous line.